### DISCUSSION TITLE
No multiple accounts possible

### DISCUSSION THREAD

**Discussion by mihaicris-adoreme** [community/community#38717] [Mobile] [answered]
<details>
<summary></summary>
GitHub for iOS, Version 1.88.0 (82868297)
Device: iPhone 12 Pro (iOS 16.1)
</details>

**Comment by wikolopadm** [comment]
Hi, 
https://github.com/https-github-com-freeq/runREGip

**Reply by mihaicris-adoreme** [reply]
Thank you, but I'm getting 404

**Comment by wikolopadm** [comment]
Hello,Please excuse me, I am new to GitHub, however I have 40 years of IT behind me, what are you referring to in your email when you say:   Thank you, but I'm getting 404 Please let me know what is the problem Laurent PERRETConsultant  andUnix engineering / .PHP / .VBS / .CMD / .REG  Envoyé à partir de Courrier pour Windows De : Mihai CristescuEnvoyé le :mercredi, 9 novembre 2022 22:58À : community/communityCc : Laurent PERRET; CommentObjet :Re: [community/community] No multiple accounts possible (Discussion #38717) Thank you, but I'm getting 404—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

**Comment by jsoref** [comment]
The are many discussions with this request. Please note that on average if you're asking, you're probably in violation of the terms of service. 
https://github.com/community/community/discussions/39693#discussioncomment-5754343

**Comment by eliperkins** [comment, marked as answer]
Hey @mihaicris-adoreme and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.

**Reply by mihaicris-adoreme** [reply]
Thanks a lot! Fantastic news!
